### ISSUE TITLE
Cask type should actually check for "available updates", not only for "is installed"

### ISSUE THREAD

**Issue by frdmn** [mattly/bork#50]
Cask doesn't handle upgrades well at the moment, and whenever you try to upgrade a specific application using cask you have to reinstall it manually to "gain" said update.

I had some troubles understanding what the actually problem was in specific so I asked in the Gitter room, so here's the conversation with @adidalal:

```
frdmn: Good morning.
frdmn: Can someone help me out understand the current behaviour of upgrading apps using Cask?
frdmn: I have quite a bunch of apps installed using cask. However, I notice a lot of them are outdated. When I check using brew cask info [app] then, it shows a newer version number than currently installed locally and also mentions Not installed.
frdmn: When I attempt to install the (already existing, but outdated) cask using brew cask install <app>, it downloads and installs the newer version besides the currently existing one.
frdmn: Now brew cask info <app> shows the specific cask as installed, but forgot to uninstall or remove the obsolte version locally. Thus, Spotlight (or Alfred) display both versions, the old and the recently updated one. (http://up.frd.mn/chhmJvOj82.png).
frdmn: Can someone shed some light for me? Am I missing something here or is Cask just not able to handle updates properly at the current state of development?
adidalal: @frdmn The latter, currently. It's a common issue/feature request. Here's (https://github.com/caskroom/homebrew-cask/issues/16033#issuecomment-166510222) a good summary of the options and alternatives currently. Specifically, this (https://github.com/bgandon/brew-cask-outdated) and this (https://github.com/buo/brew-cask-upgrade) are unofficial scripts to do what you are asking for. (PRs welcome, naturally )
frdmn: @adidalal Thanks! I'm going to give the alternative scripts a shot.
```

I thought about this and came to the conclusion that bork would be perfect for this job. 

Instead of just checking if the specific cask application is locally installed, an actual version check would be nice.

**Comment by mattly** [comment]
Thanks! I'll take a look.

**Comment by mattly** [comment]
Closed by #52
